src/types.js

```diff
@@ -404,12 +404,14 @@ export type BehaviorOptions = {|
   behaviorElement: Element,
   showIndicatorId?: string,
   delay?: number,
+  targetId: ?string,
```

Comment:
the presence of the key should be made optional, but when present, it's value should be set - you can do that by moving the `?` before `:`:
```suggestion
  targetId?: string,
```